### PR TITLE
Some Regulations and Advices, pleases!!!

### PR DESCRIPTION
**Pull request by ynskardas**

### CODE REVIEW

File: front-end/src/Views/ProfilePage.js [hunk 1]
```diff
@@ -100,7 +100,11 @@ export default class ProfilePage extends Component {
       projects: [],
       file : undefined,
       showUpload: false,
-      loading: true // For eradicating glitches due to request delays
+      loading: true,
+      isFollow: [{
+        otherProfileId: "",
```
Comment (Chuchichachtli) [inline front-end/src/Views/ProfilePage.js:105]: You can put them as seperate states. Bundling up like this is confusing

File: front-end/src/Views/ProfilePage.js [hunk 2]
```diff
@@ -608,4 +638,274 @@ export default class ProfilePage extends Component {
...
+          return false
+        }
+        else{
+          console.log("3")
+          return true
+        }
+  } 
+
+  renderFollow() {
+
```
Comment (Chuchichachtli) [inline front-end/src/Views/ProfilePage.js:673]: You shouldn't make api calls every time something renders, it's better to put it in component did mount

File: front-end/src/Views/ProfilePage.js [hunk 3]
```diff
@@ -608,4 +638,274 @@ export default class ProfilePage extends Component {
...
+              this.setState({followers: newFollowers})
+
+              console.log(this.state.followers)
+
+              //userState.following.push(profileState.profile[0].name +" "+ profileState.profile[0].middle_name +" "+ profileState.profile[0].lastname)
+              
+
+            });
+
+            const followAction = {
```
Comment (Chuchichachtli) [inline front-end/src/Views/ProfilePage.js:716]: Why did you make a const followAction ?

**Comment by liberbey** [comment]
Not relevant anymore.

**Pull request closed without merging**